addons/mail/controllers/mail.py

```diff
@@ -103,6 +103,8 @@ def _redirect_to_record(cls, model, res_id, access_token=None, **kwargs):
                     'active_id': res_id,
                     'action': record_action.get('id'),
                 }
+                if suggested_company:
```

Comment:
Could be great to add a test in (or like) ``test_redirect_to_records`` (in test_mail_thread_internals in 15, and in test_mail_multicompany in "some version").

Reply:
For sure, it's done :+1: